Assets/PatchKit Patcher/Scripts/AppData/Remote/Downloaders/ChunkedHttpDownloader.cs

```diff
@@ -77,8 +77,8 @@ private ChunkedFileStream OpenFileStream()
             }
 
             var chunksRange = CalculateContainingChunksRange(_range);
-            int startChunk = (int) (chunksRange.Start / _chunksData.ChunkSize);
-            int endChunk = _range.End == -1 ? -1 : (int) (chunksRange.End / _chunksData.ChunkSize);
+            int startChunk = (int) Math.Floor((double) chunksRange.Start / (double) _chunksData.ChunkSize);
+            int endChunk = _range.End == -1 ? -1 : (int) Math.Ceiling((double) chunksRange.End / (double) _chunksData.ChunkSize);
```

Comment:
Converting to double is not acceptable. if you'd like to find the ceiling, do a mod (%) and if greater than 0 then add ChunkSize.